Formula/kubespy.rb

```diff
@@ -0,0 +1,26 @@
+class Kubespy < Formula
+  desc "Tools for observing Kubernetes resources in realtime, powered by Pulumi"
+  homepage "https://pulumi.io/"
```

Comment:
Is there a specific homepage for this tool? I can't find any mention of `kubespy` at this domain.

Reply:
No, there's just the GitHub repo and a blog post https://blog.pulumi.com/kubespy-and-the-lifecycle-of-a-kubernetes-pod-in-four-images. Maybe https://github.com/pulumi/kubespy would be better?